src/components/DrawerTabPanels/DetailsTabPanel/DatasetTabPanel.scss

```diff
@@ -0,0 +1,3 @@
+.rdg-cell {
```

Comment:
don't think this is mine
@CursedSeraphim did you add this?
also it is called "DatasetTabPanel.scss" although it is inside the "DetailsTabPanel" folder

Reply:
if possible we can put it as a local state. can't remember whether there was a particular problem that made this impossible

but I think we also discussed whether it can be addressed solely in one element and then we decided to change it for all .rdg-cell elements for this to not become a time-dump